kornia/filters/kernels.py

```diff
@@ -80,7 +87,16 @@ def gaussian(
     KORNIA_CHECK_SHAPE(sigma, ["B", "1"])
     batch_size = sigma.shape[0]
 
-    x = (torch.arange(window_size, device=sigma.device, dtype=sigma.dtype) - window_size // 2).expand(batch_size, -1)
+    if center is None:
```

Comment:
can you add a specific case for it under kernels tests?

Reply:
There are no test for gaussian function, i assume that they are inside get_gaussian_kernel1d. 
I will have to add "mean" argument to get_gaussian_kernel1d as well, won't I? and what about get_gaussian_kernel2d and get_gaussian_kernel2d?

Reply:
ouch, we are missing some tests. I would say to just add a simple test for `gaussian`... I'm not sure if "mean" is relevant for the internal kernel generators